requirements.txt

```diff
@@ -32,7 +34,10 @@ webob
 ziggurat-foundations==0.8.3
 zope.sqlalchemy==1.1
 # TODO: remove when merged
-# until fix merged and deployed (https://github.com/authomatic/authomatic/pull/195)
-#   authomatic==0.1.0.post1
-#-e git+https://github.com/fmigneault/authomatic.git@httplib-port#egg=Authomatic
-https://github.com/fmigneault/authomatic/archive/httplib-port.zip#egg=Authomatic>=0.1.0.post2
```

Comment:
Just curious, the old broken Automatic was from your fork so you control it so it should not change, how come it suddenly becomes broken?

Reply:
My fork adds a quick fix for ssl verification bypass, but requirements where very broken in the original repo. There was a lot of hacking around to make it work.
I was waiting until the real repo fixed their stuff, which was massively delayed by owner changes and many ci tests not passing.
Now that requirements are fixed, I rebase'd on them to keep up to date, but I realized afterwards that openid requirement is still not automatically added (it's an extra). 
I didn't notice it at first as I had it already installed.